seed/csharp-sdk/alias-extends/src/SeedAliasExtends.Test/Core/RawClientTests/AdditionalParametersTests.cs

```diff
@@ -0,0 +1,300 @@
+using NUnit.Framework;
+using SeedAliasExtends.Core;
+using WireMock.Matchers;
+using WireMock.Server;
+using SystemTask = global::System.Threading.Tasks.Task;
+using WireMockRequest = WireMock.RequestBuilders.Request;
+using WireMockResponse = WireMock.ResponseBuilders.Response;
+
+namespace SeedAliasExtends.Test.Core.RawClientTests;
+
+[TestFixture]
+[Parallelizable(ParallelScope.Self)]
+public class AdditionalParametersTests
+{
+    private WireMockServer _server;
+    private HttpClient _httpClient;
+    private RawClient _rawClient;
+    private string _baseUrl;
+
+    [SetUp]
+    public void SetUp()
+    {
+        _server = WireMockServer.Start();
+        _baseUrl = _server.Url ?? "";
```

Comment:
?? throw new

Reply:
Discussed offline - this is left as-is for consistency with other tests.